pyccel/codegen/printing/fcode.py

```diff
@@ -481,7 +465,8 @@ def _print_Print(self, expr):
 
     def _print_SymbolicPrint(self, expr):
         # for every expression we will generate a print
-        _iprint = lambda e: "print *, 'sympy> {}'".format(e)
+        def _iprint(e):
+            return "print *, 'sympy> {}'".format(e)
         code = ''
         for a in expr.expr:
             code = '{code}{p}'.format(code=code, p=_iprint(a))
```

Comment:
Concatenating a string within a for loops seems a little wasteful. Maybe write
```
code = "".join(map(_iprint, expr.expr))
```
But aren't we missing a carriage return `'\n'` between statements?

Reply:
I think we probably are. But I don't know exactly what a symbolic print is. @saidctb ?